misk/src/main/kotlin/misk/web/actions/StatusActionApi.kt

```diff
@@ -0,0 +1,34 @@
+package misk.web.actions
+
+import misk.healthchecks.HealthCheck
+import misk.web.Get
+import misk.web.ResponseContentType
+import misk.web.mediatype.MediaTypes
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class StatusActionApi : WebAction {
+    @Inject lateinit var healthChecks: MutableSet<HealthCheck>
+
+    @Get("/api/_status")
```

Comment:
Should use `_status` as the endpoint, for consistency with other internal actions. Unadorned paths should be reserved for application use.